InformationScripting/scripts/sink.py

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
I agree that this is less implementation, compared to the C++ version, but if you consider my run-time argument, it suddenly feels very wasteful, don't you think?